packages/editor/src/components/entities-saved-states/index.js

```diff
@@ -210,33 +106,37 @@ export default function EntitiesSavedStates( { isOpen, close } ) {
 			</div>
 
 			<div className="entities-saved-states__text-prompt">
-				<h2>
-					{ __( 'Please review the following changes to save:' ) }
-				</h2>
+				<strong>{ __( 'Are you ready to save?' ) }</strong>
+				<p>
+					{ __(
+						'Changes were made to this template, its content, and the contents of the page.'
```

Comment:
There are a few things wrong with this copy:

- It assumes we are editing a template.
- It's redundant in saying the template and its content have changed.
- It's not clear what the contents of the page are.

Can we concatenate the list names? So we have something like:
"Changes were made to a template, posts, template parts, and a page.

Note that it takes into account single/plural.

Reply:
> It assumes we are editing a template.

Is there a situation where this would show up where the user isn't editing a template? Wouldn't they just see the normal publish flow?

> It's redundant in saying the template and its content have changed.

Fair enough

> It's not clear what the contents of the page are.

Do you have suggestions here?

> Can we concatenate the list names? So we have something like:
> "Changes were made to a template, posts, template parts, and a page.

Does this mean the sentence would change based on the entities with changes? If so, that would be great.

Reply:
> There are a few things wrong with this copy:

100% agree.

> Can we concatenate the list names?
Note that it takes into account single/plural.

Yes, I think this will be the best solution. 

<hr/>

> Is there a situation where this would show up where the user isn't editing a template? Wouldn't they just see the normal publish flow?

This can be present in any editor where multiple entities are present.  For example if you are in the post editor and have made changes to a template part contained in there, this multi entity save flow will appear.  The normal publish flow does not take the extra entities into account, as it is only concerned with the publish state/visibility for the parent entity (the page or post etc.).

> Does this mean the sentence would change based on the entities with changes? If so, that would be great.

Yup.  This should be doable.

Reply:
> The normal publish flow does not take the extra entities into account, as it is only concerned with the publish state/visibility for the parent entity (the page or post etc.).

This raises a red flag for me as a significant amount of users are going to be expecting their usual Publish flow (which can include things like publicizing their article to social networks). It might mean we need to work out how multi-entity saving works within the Pre-publish flow and also how it will appear for folks who have disabled pre-publish (likely how we have designed it currently).

Reply:
>>It's not clear what the contents of the page are.
>
>Do you have suggestions here?

We can say the page has changed, but when we say "the contents" that could include template parts and other things in some people's minds.

Also, note that reusable blocks could show up in the list of changed things.

>This raises a red flag for me as a significant amount of users are going to be expecting their usual Publish flow

If the post in question is selected for saving, we could transition into the pre-publish flow after hitting save.

Reply:
> It might mean we need to work out how multi-entity saving works within the Pre-publish flow and also how it will appear for folks who have disabled pre-publish (likely how we have designed it currently)

The pre-publish flow still works the same as it did before in the page/post editors.  If there is no multi-entity saving triggered, there is no save panel and no change observed.  If it is triggered (by editing a template part on that page), then the save panel leads into the standard publish flow (or not if disabled).

Reply:
I updated the phrase to generate based off what has been edited (including plurality):

![updating-phrase](https://user-images.githubusercontent.com/28742426/82384195-57616e00-99fd-11ea-8853-68f9a1746bf9.gif)